### PR TITLE
(anitya) Substitute Fedora package name for project name if available.

### PR DESCRIPTION
**Pull request by ralphbean**
This should fix fedora-infra/the-new-hotness#21.

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/anitya.py [hunk 1]
```diff
@@ -118,10 +118,20 @@ def subtitle(self, msg, **config):
...
             if 'message' in msg['msg']:
                 message = msg['msg']['message']
             else:
                 message = msg['msg']
+
+            # If this project is mapped to a Fedora package, then just
+            # substitute "our" package name for the project name to make
+            # everyone's emails more readable.
+            # https://github.com/fedora-infra/the-new-hotness/issues/21
+            for package in message['packages']:
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/anitya.py]: Should we use ``.get()``?

**Comment by pypingou** [comment]
One question, otherwise :+1:

File: fedmsg_meta_fedora_infrastructure/anitya.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/anitya.py]: Heh, what does anitya return?  I thought it always had at least a ``[]`` for 'packages'.  Might as well be on the safe side.

Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/anitya.py]: That would be my assumption as well, but I didn't check the code and safe_side++ :)

**Pull request merged**